
enum MSG_ACTION_TYPE
{
    MSG_REQUEST = 0;
    MSG_RESPONSE = 1;
    MSG_NOTIFY= 2;
}


enum MessageType
{
    MSG_SOCCER_PLAYER_INFO = 1;
    MSG_HEART_BEAT          = 2;//心跳
}

message SoccerPlayerInfoRequest
{
    required int32 player_id = 1;
}

message SoccerPlayerInfoResponse
{
    required int32 player_id = 1;
    required string player_name = 2;
    required int32 age = 3;
}

message ClientHeartBeatRequest
{
    required int32 client_time = 1;
}

message ClientHeartBeatResponse
{
    required int32 server_time = 1;
}
